CesiumGeospatial/test/TestBoundingRegionBuilder.cpp

```diff
@@ -17,6 +19,10 @@ TEST_CASE("BoundingRegionBuilder") {
     CHECK(!rectangle.contains(Cartographic(0.0, 1.0, 0.0)));
     CHECK(!rectangle.contains(Cartographic(0.0, -1.0, 0.0)));
 
+    // The rectangle returned by toGlobeRectangle should be identical.
+    GlobeRectangle rectangle2 = builder.toGlobeRectangle();
+    CHECK(std::memcmp(&rectangle, &rectangle2, sizeof(GlobeRectangle)) == 0);
```

Comment:
Super nitpicky, but would you object to explicitly writing each edge check instead?

```
CHECK(rectangle.getWest() == rectangle2.getWest());
```
For me, these checks would make more sense from a glance. And if you're okay with it, I don't mind replacing the lines on your behalf :smile:

(Didn't realize you could do an equality test with `std::memcmp` , that's neat!)

Reply:
I'm not opposed, just lazy. Even better than putting the individual checks in the tests would be to provide an `equals` / `equalsEpsilon` method on `GlobeRectangle` itself. `memcmp` works great for something with 4 doubles like GlobeRectangle, but isn't a general solution. Even just padding between fields of different sizes could throw it off.